.plugin_config/codestyle.xml

```diff
@@ -57,7 +57,7 @@
 
         <!-- Allow supressions per file -->
         <module name="SuppressionXpathFilter">
-            <property name="file" value=".plugin_config/codestyle_suppressions.xml"/> <!-- from project root -->
+            <property name="file" value="./target/checkstyle-suppressions.xml"/>
```

Comment:
omg

Reply:
?

Reply:
Hope you understand that the pom.xml change makes it so that *the contents of the suppressions XML is copied into the ${project.build.directory}/checkstyle-suppressions.xml file before being passed to Checkstyle for loading.*

Reply:
thats why I'm omging.... Holly symlinks...